src/main/kotlin/org/sol4k/SignIn.kt

```diff
@@ -0,0 +1,79 @@
+package org.sol4k
+
+data class SignInInput(
```

Comment:
It looks like this class was added by mistake. Or am I mistaken?

Reply:
I will remove it.